### PR TITLE
[IOTDB-3005] Increase read and write permissions in ConfigExecution

### PR DESCRIPTION
**Pull request by RYH61**
1、Removed AuthorNode related content
2、Make permissions executed through ConfigExecution in both query and write

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/Coordinator.java [hunk 1]
```diff
@@ -75,7 +74,7 @@ private IQueryExecution createQueryExecution(
       IPartitionFetcher partitionFetcher,
       ISchemaFetcher schemaFetcher) {
     if (statement instanceof ConfigStatement) {
-      queryContext.setQueryType(QueryType.WRITE);
+      queryContext.setQueryType(ConfigExecution.operateIsQuery((ConfigStatement) statement));
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/mpp/execution/Coordinator.java:77]: How about putting isQuery() into ConfigStatement

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java [hunk 2]
```diff
@@ -19,38 +19,32 @@
...
 import org.slf4j.LoggerFactory;
 
-import java.util.Locale;
-
 public class AuthorizerConfigTask implements IConfigTask {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AuthorizerConfigTask.class);
 
   private AuthorStatement authorStatement;
+  private AuthorizerManager authorizerManager = AuthorizerManager.getInstance();
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java:39]: Is using a ClusterAuthorizer enough?

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java]: fix

File: server/src/main/java/org/apache/iotdb/db/mpp/sql/statement/metadata/SetStorageGroupStatement.java [hunk 3]
```diff
@@ -44,4 +45,9 @@ public <R, C> R accept(StatementVisitor<R, C> visitor, C context) {
   public void setStorageGroupPath(PartialPath storageGroupPath) {
     this.storageGroupPath = storageGroupPath;
   }
+
+  @Override
+  public QueryType operatorIsQuery() {
```
[the remark this answers is not in the document]
Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/mpp/sql/statement/metadata/SetStorageGroupStatement.java:50]: fix

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java (hunk 2 above)
Comment (wangchao316) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java]: if use ClusterAuthorizer ， The permission class cannot be reflected from the configuration.

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java]: ok

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java:38]: > if use ClusterAuthorizer ， The permission class cannot be reflected from the configuration.

fix

**Review by qiaojialin — APPROVED** [review]
LGTM~

File: server/src/main/java/org/apache/iotdb/db/auth/authorizer/ClusterAuthorizer.java [hunk 4]
```diff
@@ -16,8 +16,95 @@
...
+      if (TSStatusCode.SUCCESS_STATUS.getStatusCode() != authorizerResp.getStatus().getCode()) {
+        LOGGER.error(
+            "Failed to execute {} in config node, status is {}.",
+            AuthorOperator.AuthorType.values()[authorizerReq.getAuthorType()]
+                .toString()
+                .toLowerCase(Locale.ROOT),
+            authorizerResp.getStatus());
+        future.setException(new StatementExecutionException(authorizerResp.getStatus()));
+      } else {
+        // TODO: Construct result
```
Comment (xingtanzjr) [inline server/src/main/java/org/apache/iotdb/db/auth/authorizer/ClusterAuthorizer.java:95]: Let's supply the detail here

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/auth/authorizer/ClusterAuthorizer.java:95]: Here I will construct a tsBlock

**Pull request merged**